src/magic_folder/magic_file.py

```diff
@@ -73,7 +73,7 @@ class MagicFileFactory(object):
     _config = attr.ib()  # MagicFolderConfig
     _tahoe_client = attr.ib()
     _folder_status = attr.ib()
-    _local_snapshot_service = attr.ib()
+    _local_snapshot_service = attr.ib()#validator=attr.validators.instance_of(LocalSnapshotService))
```

Comment:
Is this an actual pragma-like directive for `attrs`, or something commented-out (that should maybe be removed)? 

I haven't used `attrs` much, so not sure what the actual intent is here -- or why the validator is outside (and not inside) `attr.ib()`'s parens.

Reply:
there's no pragmas, that's like "note to self" (because circular import :/ )